.github/workflows/ci.yml

```
@@ -16,25 +16,84 @@ concurrency:
 
 jobs:
   build_and_test:
-    name: Build and test bosdyn_msgs packages
-    runs-on:
-      group: bdai-gh-large-runners
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - { runner: "ubuntu-22-04-large-runner", image: "ros:humble-ros-base-jammy", arch: "AMD64" }
+        - { runner: "k8s-t2a-standard-16", image: "arm64v8/ros:humble-ros-base-jammy", arch: "ARM64" }
```

@amessing-bdai nit: 
```suggestion
        - { runner: "ubuntu-22-04-large-runner", image: "ros:humble-ros-base-jammy", arch: "amd64" }
        - { runner: "k8s-t2a-standard-16", image: "arm64v8/ros:humble-ros-base-jammy", arch: "arm64" }
```
just for the sake of consistency with CD jobs